PocketKit/Sources/Localization/Strings.swift

```diff
@@ -725,6 +725,12 @@ public enum Localization {
         public static let title = Localization.tr("Localizable", "tips.recommendationsWidget.selectTopic.title", fallback: "New Recommendations Widget!")
       }
     }
+    public enum SharedWithYouActions {
+      /// Tap the attribution view to reply, long press it to remove an item or reply.
+      public static let message = Localization.tr("Localizable", "tips.sharedWithYouActions.message", fallback: "Tap the attribution view to reply, long press it to remove an item or reply.")
```

Comment:
Can we change this to "Tap here to reply to this message or long press to remove this item."
Can we remove the title?

Reply:
@Gio2018 Ok, keep the title just change the description, thanks!

Reply:
Here's what I mean:

<p align=center>
<img width=320 src=https://github.com/Pocket/pocket-ios/assets/34376330/7ca5492a-f5f5-4a04-87fd-709d13f768bf>
</p>

Reply:
ok changes made! Here's the final version

<p align=center>
<img width=320 src=https://github.com/Pocket/pocket-ios/assets/34376330/b83a1dbe-e742-4312-8721-718d9daea8bd>

<p>